api/src/main/java/com/ford/labs/retroquest/columntitle/ColumnTitleService.java

```diff
@@ -0,0 +1,42 @@
+package com.ford.labs.retroquest.columntitle;
+
+import com.ford.labs.retroquest.exception.ColumnTitleNotFoundException;
+import com.ford.labs.retroquest.websocket.WebsocketColumnTitleEvent;
+import com.ford.labs.retroquest.websocket.WebsocketEventType;
+import com.ford.labs.retroquest.websocket.WebsocketService;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ColumnTitleService {
+
+    private final ColumnTitleRepository columnTitleRepository;
+    private final MeterRegistry meterRegistry;
+    private final WebsocketService websocketService;
+
+    public ColumnTitleService(ColumnTitleRepository columnTitleRepository, MeterRegistry meterRegistry, WebsocketService websocketService) {
+        this.columnTitleRepository = columnTitleRepository;
+        this.meterRegistry = meterRegistry;
+        this.websocketService = websocketService;
+    }
+
+    public List<ColumnTitle> getColumnTitlesByTeamId(String teamId) {
+        return columnTitleRepository.findAllByTeamId(teamId);
+    }
+
+    public ColumnTitle editColumnTitleName(Long columnId, String newColumnName, String teamId) {
+        var existingColumnTitle = columnTitleRepository.findById(columnId).orElseThrow(ColumnTitleNotFoundException::new);
+        existingColumnTitle.setTitle(newColumnName);
+
+        ColumnTitle newColumnTitle = columnTitleRepository.save(existingColumnTitle);
+
+        websocketService.publishEvent(new WebsocketColumnTitleEvent(teamId, WebsocketEventType.UPDATE, newColumnTitle));
+
+        meterRegistry.counter("retroquest.columns.changed.count").increment();
```

Comment:
I know there's an `@Timer` annotation that can be added to a method to get both count and timing metrics for a function. I wonder if there's also one for the counter? Unless we specifically only want to count the number of successful change requests and not the number of change requests total.

Reply:
I guess we could count every attempt to change a column title? We're just looking to understand whether folks change the titles more often or not after we publish some information about retrospectives, not add instrumentation looking for failure cases or anything.

Reply:
Alright, this looks good enough for now then.